.github/workflows/dependabot-approve-merge.yml

```diff
@@ -31,20 +31,13 @@ jobs:
       pull-requests: write
 
     steps:
-      - uses: mdecoleman/pr-branch-name@bab4c71506bcd299fb350af63bb8e53f2940a599 # v2.0.0
-        id: branchname
-        with:
-          repo-token: ${{ secrets.GITHUB_TOKEN }}
-
-      # Github actions bot approve
+      # GitHub actions bot approve
       - uses: hmarr/auto-approve-action@b40d6c9ed2fa10c9a2749eca7eb004418a705501 # v2
-        if: contains(steps.branchname.outputs.branch, '/webrtc-adapter-') != true && contains(steps.branchname.outputs.branch, '/nextcloud/vue-') != true
```

Comment:
cc @nickvergessen 
this was intended from our side, IIRC

Reply:
yeah, just revert it and push